app/helpers/money_input_helper.rb

```diff
@@ -0,0 +1,10 @@
+module MoneyInputHelper
+  def format_money_input(log:, question:)
+    value = log[question.id]
+
+    return unless value
+    return value unless question.prefix == "£"
+
+    sprintf("%.2f", value)
```

Comment:
```suggestion
    number_with_precision(value, precision: 2, delimiter: ',')
```

Reply:
maybe even https://apidock.com/rails/ActionView/Helpers/NumberHelper/number_to_currency

Reply:
that's the method being used in `log:format_as_currency` but I dunnae think he wants the £ prefix here

Reply:
yep, think the unit is optional if you give it ""